.github/workflows/bash.yml

```diff
@@ -19,8 +19,8 @@ jobs:
 
       - name: Build Docker container
         run: |
-          docker build -t bash:test dockerfiles/bash/
-          docker run --name bash -v $(pwd -P):/root/cylc-flow --rm -t -d bash:test /bin/bash
+          docker build -t bash:test  -m 1G --memory-swap 1G dockerfiles/bash/
+          docker run --name bash -v $(pwd -P):/root/cylc-flow --rm -t -d -m 1G --memory-swap 1G bash:test /bin/bash
```

Comment:
Was getting some errors due to low virtual memory when building the container. Not sure if others have seen this, but might be good to constraint resources so our builds don't fail sporadically.